appveyor/install.ps1

```diff
@@ -162,7 +162,8 @@ function InstallPip ($python_home) {
         $webclient = New-Object System.Net.WebClient
         $webclient.DownloadFile($GET_PIP_URL, $GET_PIP_PATH)
         Write-Host "Executing:" $python_path $GET_PIP_PATH
-        Start-Process -FilePath "$python_path" -ArgumentList "$GET_PIP_PATH" -Wait -Passthru
+        Get-ChildItem "$GET_PIP_PATH" -force
```

Comment:
Yeah, sorry, the pip failures are going to require some awkward frequent-commit debugging...  I guess I can do that in coverage.py repo actually....